.github/workflows/IJ.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Validate against IJ versions
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        IJ: [IC-2020.1, IC-2020.2, IC-2020.3]
```

Comment:
Why 2019.3 has been removed ?

Reply:
you wrote it badly before https://github.com/redhat-developer/intellij-redhat-telemetry/pull/2#discussion_r584969563 :laughing:

Reply:
That was a test 🥰

Reply:
I had noticed this contradiction. I was thinking that you had done this on purpose because I require "193.5233.102" and it would thus not compile with the base version  IC-2019.3. See [2019.3](here https://github.com/JetBrains/intellij-community/tree/193) vs [193.5233](https://github.com/JetBrains/intellij-community/tree/193.5233). Wrong?

Reply:
Yes but we should revert then gradle.properties and plugin.xml to 2019.3 and add it there then

Reply:
added 2019.3 to IJ.yaml